packages/website/ts/pages/wiki/wiki.tsx

```diff
@@ -95,6 +95,7 @@ export class Wiki extends React.Component<WikiProps, WikiState> {
                     menuSubsectionsBySection={menuSubsectionsBySection}
                     translate={this.props.translate}
                     sidebarHeader={sidebarHeader}
+                    style={{ zIndex: 1 }}
```

Comment:
this also needs to be fixed in the documentation page, can we just put this directly into the `TopBar` component?